src/components/Books.js

```diff
@@ -1,35 +1,42 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import BookItem from './BookItem';
+import { addBook } from '../redux/books/booksSlice';
+/* eslint-disable react/jsx-props-no-spreading */
```

Comment:
- [ ] Please make sure that never disable the eslinter in your code, try to understand why are you getting the eslinter for that specific line or file. Because it is also against the general requirements of the project. Kindly remove it and try to resolve the issue without disabling linters.